### DISCUSSION TITLE
Enable SSR for Demo app? For support SEO purpose ^^

### DISCUSSION THREAD

**Discussion by hienhuynhtm** [realworld-apps/realworld#1430] [Community]
Hey guys,

Thanks you guys so much for very great app.
But do we have any quick guide to enable SSR to support SEO for our Demo app?
Many thanks,
Andy.

**Comment by hienhuynhtm** [comment]
I found this one when searching: https://codesandbox.io/s/arrlancorenextjs-ssr-real-world-app-example-w2gr6?file=/pages/index.js
Is it good for us?

**Comment by hienhuynhtm** [comment]
I found we have 2 repos which support SSR. Which one is better to use @geromegrignon ?
1. https://github.com/ranwahle/realworld-qwik-app
2. https://github.com/gang-of-front/realworld-qwik

**Comment by geromegrignon** [comment]
Hi @hienhuynhtm, I thought about it for a while (well years). But it requires a server and given the project's popularity, it might quickly go above the free tier of existing providers. As it's a free project without any sponsors, that's the limit I don't want to cross for my own sake :)
